Cargo.toml

```diff
@@ -12,24 +12,36 @@ license = "MIT OR Apache-2.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+cli = ["dep:needletail", "dep:clap", "dep:rayon", "dep:log", "dep:stderrlog"]
+
+[[bin]]
+name = "sablast"
+path = "src/cli/main.rs"
+required-features = ["cli"]
+
 [dependencies]
 ## core
-# TODO Re-enable reading compressed sequences in needletail
-# This requires resolving the libllzma linker issue in build_artifacts.yml
-needletail = { version = "0.5.1", default-features = false }
-rayon = "1"
 sbwt = "0.3.1"
 
 ## cli
-clap = { version = "4", features = ["derive"] }
+clap = { version = "4", features = ["derive"], optional = true}
+
+### io
+#### TODO Re-enable reading compressed sequences in needletail
```

Comment:
A new Todo was discovered. If it is not a priority right now,consider marking it for later attention.
TODO Re-enable reading compressed sequences in needletail
- [ ] Ignore